ibm/service/database/resource_ibm_database.go

```diff
@@ -467,6 +467,29 @@ func ResourceIBMDatabaseInstance() *schema.Resource {
 					},
 				},
 			},
+			"logical_replication_slot": {
```

Comment:
This should have validation to enforce only `postgresql` deployments can use `logical_replication_slot`

Reply:
Thanks! Took your recommendation and added it here: https://github.com/IBM-Cloud/terraform-provider-ibm/blob/7a2fac09b90afcaa3d928c53f647e09eb11801b7/ibm/service/database/resource_ibm_database.go#L2301 :D

Reply:
I think we might want to do this in a `CustomizeDiff` function if we can't do it as a validation function since this will only blow up during runtime rather than during the plan stage

Reply:
Done :D